styles/modules/_form.scss

```diff
@@ -0,0 +1,42 @@
+.form {
+    &--compact {
+        @include mq($screen-s) {
+            display: flex;
+            justify-content: center;
+        }
+        .form__title {
+            line-height: $button-height;
+            margin-right: 20px;
+        }
+        .form__content {
+            display: flex;
+        }
+        .input {
+            margin-bottom: 0;
+            width: 100%;
+            @include mq($screen-s) {
+                display: flex;
+                justify-content: center;
+            }
+        }
+        .input__field {
+            width: 100%;
+            border: $button-icon-border;
+            line-height: $button-height;
+            height: $button-height;
+            border-radius: 0;
+            @include mq($screen-s) {
+                width: 290px;
+            }
+        }
+        .button {
+            margin: 0 0 0 -3px;
+        }
+    }
+
+    div.mage-error[generated] {
+        margin-top: 7px;
+        color: $color-danger;
+        font-size: $font-size-extra-small;
+    }
+}
```

Comment:
good question. I'd update the component form with modifier but validation part I'd move to theme styles. @Igloczek what do you think?